.github/workflows/ci.yml

```
@@ -156,16 +157,34 @@ jobs:
           key: nginx-kubernetes-gateway-${{ github.run_id }}-${{ github.run_number }}
       - name: Docker Buildx
         uses: docker/setup-buildx-action@v1
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        if: ${{ startsWith(github.ref, 'refs/tags') }}
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            name=ghcr.io/pammecrandall/nginx-kubernetes-gateway,enable=${{ startsWith(github.ref, 'refs/tags') }}
+            name=nginx/nginx-kubernetes-gateway,enable=${{ startsWith(github.ref, 'refs/heads') }}
+          tags: |
+            type=semver,pattern={{version}},enable=${{ startsWith(github.ref, 'refs/tags') }}
```

```suggestion
            type=semver,pattern={{version}}
```
You don't need this, it's already how it works, it's just used on a push of a tag

---

Sure.  I can remove it.

---

The enable attribute has been deleted.